src/depot_type.h

```diff
@@ -10,9 +10,13 @@
 #ifndef DEPOT_TYPE_H
 #define DEPOT_TYPE_H
 
-typedef uint16 DepotID; ///< Type for the unique identifier of depots.
+#include "station_type.h"
+
+typedef StationID DepotID; ///< Type for the unique identifier of depots.
```

Comment:
This is a bit late I know, but this seems misleading from a type point of view.

A DepotID is not actually a StationID even if they happen to have the same field width.
i.e. values of type DepotID and StationID are not interchangeable, and they do not point into the same pools.